sdk/version.json

```diff
@@ -0,0 +1,7 @@
+{
+    "core": "0.7.1-dev",
```

Comment:
Should be `0.7.2-dev` as `0.7.1` is already released. In the next `0.7.2` release, the `-dev` will be removed